### PR TITLE
Update author

### PR DESCRIPTION
**Pull request by janiversen**
Not a big deal,

but I think that being the third overall most active contributor (https://github.com/home-assistant-libs/pytradfri/graphs/contributors ) makes this PR fair.

### CODE REVIEW

**Review by ggravlingen — APPROVED** [review]

**Comment by ggravlingen** [comment]
Definitely agree, thanks for all your improvements to the library!

**Comment by janiversen** [comment]
Thanks, more to come, (like converting raw to a dataclass), but I miss a place to get feedback on my ideas. Do you think making such “discussions” could be done in an issue ?

**Comment by ggravlingen** [comment]
I took the liberty of adding you as a maintainer to Pypi as well.

**Comment by ggravlingen** [comment]
I've seen there's a discussions feature here in Github. Maybe we should introduce that?

**Comment by janiversen** [comment]
To be honest I have not tried that, but I am totally open….important for me is to get a second opinion on basic changes, before I implement them.

**Comment by ggravlingen** [comment]
This library is part of the HA echosystem so maybe @MartinHjelmare can provide some insight on how this is normally done in the HA organisation?

**Comment by janiversen** [comment]
Yes it would be nice to hear the opinion of @MartinHjelmare 

I just looked at the documentation:
https://docs.github.com/en/discussions/quickstart#enabling-github-discussions-on-your-repository
and (again( it demands superpowers, which I assume you have.

**Comment by MartinHjelmare** [comment]
We think it's ok to just keep using issues for discussion in this repo. In lower traffic repos activating discussions may cause more confusion than help.

**Comment by ggravlingen** [comment]
Thanks for the clarification!

**Pull request merged**
